src/app/GitExtUtils/GitUI/JoinableTaskScope.cs

```diff
@@ -0,0 +1,31 @@
+﻿using GitUI;
+using Microsoft.VisualStudio.Threading;
+
+namespace GitExtUtils.GitUI;
+
+public sealed class JoinableTaskScope : IDisposable
```

Comment:
This doesn't appear used within GitUI, but rather used in tests. Should we move it there?